daemons/dss-checkout-sfn/app.py

```diff
@@ -23,12 +24,29 @@
 dss.Config.set_config(dss.BucketConfig.NORMAL)
 email_sender = dss.Config.get_notification_email()
 default_checkout_bucket = dss.Config.get_s3_checkout_bucket()
-
 replica = "aws"
+DEBUG = False
```

Comment:
Not clear what the value of doing this.

Reply:
"util"  was a wrong place for the state machine definition. I moved the stepfunction code for checkout service under dss/stepfunctions